js/script.js

```diff
@@ -0,0 +1,126 @@
+var pickElem = document.getElementById('js-playerPickElement');
+var resultsElem = document.getElementById('js-resultsTableElement');
+
+function setGameElements() {
+  switch(gameState) {
+    case 'started':
+        newGameElem.style.display = 'none';
+        pickElem.style.display = 'block';
+        resultsElem.style.display = 'block';
+      break;
+    case 'ended':
+        newGameBtn.innerText = 'Jeszcze raz';
+    case 'notStarted':
+    default:
+        newGameElem.style.display = 'block';
+        pickElem.style.display = 'none';
+        resultsElem.style.display = 'none';
+  }
+}
+
+setGameElements() 
+
+var playerPointsElem = document.getElementById('js-playerPoints'),
+    playerNameElem = document.getElementById('js-playerName'),
+    computerPointsElem = document.getElementById('js-computerPoints');
+
+function newGame() {
+	player.name = prompt('Graczu, wpisz swoje imię', 'imię gracza');
+	if (player.name) {
+		player.score = computer.score = 0;
+		gameState = 'started';
+		setGameElements();
+	playerNameElem.innerHTML = player.name;
+    setGamePoints(); // ta funkcja jeszcze nie powstała
+  }
+}
+function playerPick(playerPick) {
+    console.log(playerPick);
+}
+
+function getComputerPick() {
+	var possiblePicks = ['rock', 'paper', 'scissors'];
+	return possiblePicks[Math.floor(Math.random()*3)];
+}
+
+var playerPickElem = document.getElementById('js-playerPick');
+var computerPickElem = document.getElementById('js-computerPick');
+var playerResultElem = document.getElementById('js-playerResult');
+var computerResultElem = document.getElementById('js-computerResult');
+
+function playerPick(playerPick) {
+	var computerPick = getComputerPick();
+    
+	playerPickElem.innerHTML = playerPick;
+	computerPickElem.innerHTML = computerPick;
+}
+
+function checkRoundWinner(playerPick, computerPick) {
+  playerResultElem.innerHTML = computerResultElem.innerHTML = '';
+
+  var winnerIs = 'player';
+
+    if (playerPick == computerPick) {
+        winnerIs = 'noone'; // remis
+    } else if (
+        (computerPick == 'rock' &&  playerPick == 'scissors') ||
+        (computerPick == 'scissors' &&  playerPick == 'paper') ||
+        (computerPick == 'paper' &&  playerPick == 'rock') ) {
+        
+        winnerIs = 'computer';
+    }
+
+    if (winnerIs == 'player') {
+        playerResultElem.innerHTML = "Wygrana!";
+        player.score++;
+    } else if (winnerIs == 'computer') {
+        computerResultElem.innerHTML = "Wygrana!";
+        computer.score++;
+    }
+
+}
+
+function playerPick(playerPick) {
+    var computerPick = getComputerPick();
+    
+    playerPickElem.innerHTML = playerPick;
+    computerPickElem.innerHTML = computerPick;
+    
+    checkRoundWinner(playerPick, computerPick);
+}
+
+function setGamePoints() {
+    playerPointsElem.innerHTML = player.score;
+    computerPointsElem.innerHTML = computer.score;
+}
+
+function maxPoint () {
+    if (player.score = 10) {
```

Comment:
czy kiedykolwiek widujesz alert `PRZEGRANA` ? Nie sądzę, bo zamiast sprawdzać, czy `player.score` jest równy 10, Ty go tutaj ustawiasz na 10 i sprawdzasz czy jest on truthy -> a 10 jest tzw. `truthy` value, więc zawsze ten warunek będzie spełniony

PS. falsy value to null/undefined/0, truthy to z drobnymi wyjątkami cała reszta

PS2. Ty tej funkcji tak jak teraz patrze nawet nigdzie nie wywołujesz